addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -13,10 +32,10 @@ options.registry.WebsiteEvent.include({
      */
     allowRoomCreation(previewMode, widgetValue, params) {
         this._rpc({
-            model: this.modelName,
+            model: "event.event",
```

Comment:
Would omit this if not necessary

Reply:
It is necessary, as this.modelName is undefined.

I checked elsewhere and it seems to usually be set from data.model but that field does not exist 
![image](https://github.com/odoo/odoo/assets/18171034/fa881b4e-b804-4ca4-9fd1-4307ddc81376)

Or state.model, but state does not exist. And I could not find the model elsewhere. So, it looks like that change is necessary.

Reply:
I believe you might have gotten it from `this.options.model` but either way it requires a change, it's fine like this thanks for having a look :)